src/main/java/com/sprint/mission/discodeit/entity/User.java

```diff
@@ -0,0 +1,62 @@
+package com.sprint.mission.discodeit.entity;
+
+
+import java.util.UUID;
+import java.io.Serializable;
+import java.util.Objects;
+
+
+public class User implements Serializable  {
+    private static final long serialVersionUID = 1L;
+
+    private UUID Id;
```

Comment:
필드명은 보통 소문자로 시작해야 합니다.
```suggestion
    private UUID id;
```